packages/sync-actions/test/discount-codes-sync.spec.js

```diff
@@ -269,4 +269,68 @@ describe('Actions', () => {
     const actual = discountCodesSync.buildActions(now, before)
     expect(actual).toEqual(expected)
   })
+
+  describe('custom fields', () => {
```

Comment:
These are "thin" integration test to the otherwise thoroughly tested module to generate the custom fields. Just so that when somebody removes them from the syncer she/he gets a more descriptive error than a e.g. a mock not being called.